### PR TITLE
Post and homepage format

### PR DESCRIPTION
**Pull request by dannehdan**

### CODE REVIEW

**Review by pzrsa — COMMENTED** [review]
world record approval speed

File: app.js [hunk 1]
```diff
@@ -26,73 +26,75 @@ app.use(cookieParser());
...
-        key: "user_sid",
-        secret: "super_secret",
-        resave: false,
-        saveUninitialized: false,
-        cookie: {
-            expires: 600000,
-        },
-    })
+  session({
+    key: "user_sid",
```
Comment (pzrsa) [inline app.js:37]: is this a typo for user_id?

**Review by frontendowl — CHANGES_REQUESTED** [review]
fix a home_page_spec test, please

File: app.js (hunk 1 above)
Comment (frontendowl) [reply app.js:37]: it was like that in the template :) 'sid' may be for "secret identifier"

Comment (Anmolbhati30) [reply app.js:37]: This is the correct answer I believe

File: views/home/index.hbs [hunk 2]
```diff
@@ -9,4 +9,29 @@
     <button class="signup-button" type="button" onclick="location.href = '/sessions/new';">Login</button>
   {{/if}}
   </main>
-</div>
+</div> --}}
+
+<div class="hero-image" style="height: 100%;">
+  <div class="hero-text">
+    <h1>Welcome to {{title}}</h1>
```
Comment (frontendowl) [inline views/home/index.hbs:16]: please, add `class="title"` here, or change the test

Comment (Anmolbhati30) [reply views/home/index.hbs:16]: the dot in 'title' shouldn't be there, that's checking the page title, I'll fix that

**Review by Anmolbhati30 — APPROVED** [review]

**Pull request merged**